LayoutTests/fast/forms/ValidityState-gc.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<body>
+<p>This tests that ValidityState survives garbage collection.</p>
```

Comment:
Are there other tests that check expando properties? Does this test use the same idiom they do?

Reply:
There are various tests that test expand properties and each test has its own style / pattern. I guess having a diverse way of testing it might be a good thing since we never know what kind of subtle regression we'd end up introducing in this area. More ways we test this, more likely we'd catch a new regression.